README-win.md

```diff
@@ -0,0 +1,84 @@
+#netdisco-docker on windows10
+
+## requirements
+
+* windows 10 x64, version 1903
+  * nested virtualization needs to be enabled in the bios
+  * the following features need to be enabled in windows:
+    * containers
+    * hyper-v
+    * virtual machine platform
+    * windows hypervisor platform 
+* docker version 19.03.0-rc2
+  * docker must be set to use linux containers
+* docker-compose version 1.24.0
+
+## installing netdisco-docker on windows
+
+unlike linux we use local volumes on windows, this takes care of differences in
+userid/acl handling.
+
+* first create local volumes for netdisco's storage
+```
+    docker volume create -d local nd-pgdata-volume
+    docker volume create -d local nd-sitelocal-volume
+    docker volume create -d local nd-config-volume
+    docker volume create -d local nd-logs-volume
+```
+* fetch the windows specific compose file:
+```
+    curl -Ls -o docker-compose.yml https://raw.githubusercontent.com/netdisco/netdisco-docker/master/docker-compose-win.yml
+    docker-compose up
+```
+
+## starting/stopping containers after the initial install can be done with:
+
+* find out container id, add --all to also show stopped containers.
+```
+    docker ps
```

Comment:
May wan to say `docker ps --all` in case the containers are stopped already

Reply:
like the note on line 36 says 😄

Reply:
fair enough, though it's probably simpler to just list the full command

Reply:
well, just adding --all is easier i guess.
https://github.com/inphobia/netdisco-docker/commit/58a85bfb2185a6c30394f9b3e0a9d71b7f31a636